lib/Datastore.js

```diff
@@ -170,6 +170,29 @@ class Datastore {
         return this._client.pipeline(cmds).exec(cb);
     }
 
+    /**
+    * execute a list of commands as transaction
+    * @param {string[]} cmds - list of commands
+    * @param {callback} cb - callback
+    * @return {undefined}
+    */
+    multi(cmds, cb) {
+        return this._client.multi(cmds).exec((err, res) => {
+            if (err) {
+                return cb(err);
+            }
+            const flattenRes = [];
+            const resErr = res.find(item => {
+                flattenRes.push(item[1]);
+                return item[0] !== null;
+            });
+            if (resErr) {
+                return cb(resErr);
```

Comment:
Is `resErr` an array here? Looks like we'd need to access the error with `resErr[0]`.

Reply:
True, I forgot about that.